app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -88,3 +88,6 @@ CONFIG_CLOCK_CONTROL=y
 CONFIG_DEBUG_COREDUMP=y
```

Comment:
The commit prefix should be fixed. We haven't been very consistent in "sof/app/overlays" but "app: overlay: Enable probe MTL" would do.